web-components/src/components/modal/index.tsx

```diff
@@ -44,13 +44,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     maxWidth: theme.spacing(150),
     [theme.breakpoints.up('md')]: {
       width: '50%',
-      height: 'auto',
-      maxHeight: '90%',
+      height: '90%',
     },
     [theme.breakpoints.between('sm', 'md')]: {
       width: '70%',
-      height: 'auto',
-      maxHeight: '90%',
+      height: '90%',
     },
```

Comment:
This still lets you override the height if you want it set to 'auto' or smaller on a case-basis.

Reply:
wow, sorry, I made that change a few days ago with the forms. I thought that by default they would be better adjusted automatically. In my case I had a form that was initially small and looked too empty modal underneath. I directly changed the default behavior. I take note to do the fix and overwrite from the form.